pkg/convert/convert.go

```diff
@@ -76,91 +77,229 @@ func RegionsCsvToResources(resourcesFile string) ([]map[string]interface{}, erro
 	return finalResources, nil
 }
 
-func ResourcesToYamlRegions(resources []map[string]interface{}, outputFile string) error {
+func ResourcesToYamlRegions(resources []map[string]interface{}, outputFile string, appInterfaceDir string) error {
 
-	// Create output file if 1) it does not exist and 2) if the file is intended to be placed there
-	if strings.HasPrefix(outputFile, "output/") {
-		err := os.Mkdir("output", 0777)
-		if err != nil && !strings.Contains(err.Error(), "mkdir output: file exists") {
-			return err
+	if appInterfaceDir == "" {
+		// Create output file if 1) it does not exist and 2) if the file is intended to be placed there
+		if strings.HasPrefix(outputFile, "output/") {
+			err := os.Mkdir("output", 0777)
+			if err != nil && !strings.Contains(err.Error(), "mkdir output: file exists") {
+				return err
+			}
 		}
-	}
 
-	file, err := os.Create(outputFile)
-	if err != nil {
-		return err
-	}
+		file, err := os.Create(outputFile)
+		if err != nil {
+			return err
+		}
 
-	// Regions
-	_, err = file.Write([]byte("---\napiVersion: v1\nkind: ConfigMap\nmetadata:\n  " +
-		"annotations:\n    " + "qontract.recycle: \"true\"\n  " + "name: cloud-resources-config\ndata:\n  cloud-regions.yaml: |\n    cloud_regions:\n"))
-	if err != nil {
-		panic(err)
-	}
-	for i, resource := range resources {
-		if i == 0 {
-			continue
-		}
-		// Create block for this region
-		id := helper.AssignStringValue(resource["id"])
-		regionString := []byte(fmt.Sprintf(
-			"    - id: %s\n"+
-				"      cloud_provider_id: %s\n"+
-				"      display_name: %s\n"+
-				"      supports_multi_az: %s\n",
-			id, resource[CloudProviderId].(string), resource[DisplayName].(string), resource[SupportsMultiAz].(string)))
-		_, err = file.Write(regionString)
+		// Regions
+		_, err = file.Write([]byte("---\napiVersion: v1\nkind: ConfigMap\nmetadata:\n  " +
+			"annotations:\n    " + "qontract.recycle: \"true\"\n  " + "name: cloud-resources-config\ndata:\n  cloud-regions.yaml: |\n    cloud_regions:\n"))
 		if err != nil {
 			panic(err)
 		}
+		for i, resource := range resources {
+			if i == 0 {
+				continue
+			}
+			// Create block for this region
+			id := helper.AssignStringValue(resource["id"])
+			regionString := []byte(fmt.Sprintf(
+				"    - id: %s\n"+
+					"      cloud_provider_id: %s\n"+
+					"      display_name: %s\n"+
+					"      supports_multi_az: %s\n",
+				id, resource[CloudProviderId].(string), resource[DisplayName].(string), resource[SupportsMultiAz].(string)))
+			_, err = file.Write(regionString)
+			if err != nil {
+				panic(err)
+			}
+		}
+		file.Close()
+	} else {
+		if appInterfaceDir[len(appInterfaceDir)-1] == '/' {
```

Comment:
I think we could use strings.HasSuffix here
```suggestion
		if strings.HasSuffix(appInterfaceDir, "/") {
```